.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,26 @@
+### Description of Issue/Question
+
+### Setup
```

Comment:
Here I would expect a very brief description of the environment (from where are you trying to connect, how etc.). Do you think this would be useful for us?